code/datums/elements/embed.dm

```diff
@@ -71,8 +71,16 @@
 	if(!istype(victim) || HAS_TRAIT(victim, TRAIT_PIERCEIMMUNE))
 		return
 
+	var/flying_speed = throwingdatum?.speed || weapon.throw_speed
+
+	if(!forced && (flying_speed < EMBED_THROWSPEED_THRESHOLD && !ignore_throwspeed_threshold)) // check if it's a forced embed, and if not, if it's going fast enough to proc embedding
+		return
```

Comment:
I believe the `or` makes flying_speed a bool variable

```suggestion
	var/flying_speed = throwingdatum ? throwingdatum.speed : weapon.throw_speed

	if(!forced && (flying_speed < EMBED_THROWSPEED_THRESHOLD && !ignore_throwspeed_threshold)) // check if it's a forced embed, and if not, if it's going fast enough to proc embedding
		return
```

Reply:
no, `?.` is an operator on it's own